src/Http/LalamoveRequest.php

```diff
@@ -140,10 +154,43 @@ public function getHeaders()
         ];
     }
 
+    private function getV3Headers(): array
+    {
+        $secretKey = $this->settings->apiSecret;
+        $country    = $this->settings->country;
+
+        $requestTime = $this->clock->getCurrentTimeInMilliseconds();
+
+        $uuid = $this->uuid->getUuid();
+        $uri  = str_replace($this->settings->host, '', $this->getFullPath());
+
+        $body    = json_encode($this->getParams());
+        $message = "{$requestTime}\r\n{$this->method}\r\n{$uri}\r\n\r\n";
+
+        if ($this->method != 'GET') {
+            $message .= $body;
+        }
+
+        $publicKey = $this->settings->apiKey;
+
+        $signature = hash_hmac('sha256', $message, $secretKey);
+        $headers = [
+            // Regex:
+            // /hmac ([A-Fa-f\d]{32}|(pk_test_|pk_prod_)[A-Fa-f\d]{32}):(\d{13}):([A-Fa-f\d]{64})/
+            'Authorization' => "hmac {$publicKey}:{$requestTime}:{$signature}",
+            'Accept' => 'application/json',
+            'Content-type' => 'application/json',
+            'Market' => strtoupper($country),
+            'Request-ID' => $uuid,
+        ];
+
+        return $headers;
+    }
+
     /**
-     * @return \Lalamove\Client\Settings
+     * @todo: replace typing
```

Comment:
Action this `@todo`.

Can typehint `Lalamove\Client\V2\Settings|Lalamove\Client\V3\Settings`